agent/newrelic-install.sh

```diff
@@ -852,13 +852,13 @@ disp_get_php_list() {
 
 set_osdifile() {
   osdifile=
-  if [ -n "${NR_INSTALL_INITFILE}" ]; then
-    osdifile="${NR_INSTALL_INITFILE}"
+  if [ -n "${NR_INSTALL_INITSCRIPT}" ]; then
+    osdifile="${NR_INSTALL_INITSCRIPT}"
   fi
   if [ "${ostype}" = "darwin" ]; then
     : ${osdifile:=/usr/bin/newrelic-daemon-service}
-  elif [ "${ostype}" = "freebsd" -o -f /etc/arch-release ]; then
-    # It is possible that this is only for freebsd.
+  elif [ -f /etc/arch-release -o -d /etc/rc.d ]; then
```

Comment:
#913 addresses this problem in a more universal way applicable to wider group of OSs.